documentation/prerequisites.md

```diff
@@ -4,8 +4,14 @@ title: Prerequisites
 
 Following is the list of prerequisites for running Kubernetes with vSphere Cloud Provider:
 
-* We recommend Kubernetes version 1.6.5 and vSphere version 6.0.x
-* We support Photon, Ubuntu, CoreOs and Centos. it has been tested on Photon 1.0 GA, Ubuntu 16.04, CoreOS 4.11.2 and Centos 7.3
+* Minimum Kubernetes version version 1.6.5 
+* vSphere version 6.0.x
```

Comment:
1. In k8s and vsphere version description, can we please add a dash between version & the number ?
2. I assume we list the minimum version for everything but we support newer versions too, correct?
Example : 
vSphere version - 6.0.x onwards
RHEL OS -  7.4 onwards

Reply:
We can't assume that newer versions will be compatible

Reply:
Valid point. I guess my statement is true only in the context of k8s releases (which we actively test), correct?

Reply:
We can do that kubernetes version. Since we already minimum k8s version I think that will be fine.